Formula/wine.rb

```diff
@@ -41,14 +42,7 @@ class Wine < Formula
     end
   end
 
-  if MacOS.version >= :el_capitan
-    option "without-win64", "Build without 64-bit support"
-    depends_on :xcode => ["8.0", :build] if build.with? "win64"
-  end
-
-  # Wine will build both the Mac and the X11 driver by default, and you can switch
-  # between them. But if you really want to build without X11, you can.
-  depends_on :x11 => :recommended
+  depends_on :xcode => ["8.0", :build] if MacOS.version < :sierra
```

Comment:
Remove `if MacOS.version < :sierra`

Reply:
It builds normally without Xcode. Should this line be removed entirely?

Reply:
OK then…

Reply:
From https://github.com/Homebrew/homebrew-core/pull/6742 the point was that at least Xcode 8 is required for building with 64-bit support apparently. So we need a substitute if Xcode itself isn't required
```
depends_on :macos => :el_capitan
```

Reply:
Thanks for looking into that more closely.